roles/core/nfs_client/vars/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
-nfs_client_role_version: 1.0.3
+nfs_client_role_version: 1.0.4
+
+mount_exported_dirs: true # true=mounted, false=present
```

Comment:
Please use prefix `nfs_client_` for variables in this role. If this is a user variable, please move it to `defaults/main.yml`. Variables in `vars/main.yml` are internal to the roles.

Also, instead of a boolean, I would allow to define any state supported by the module `mount`. Is there a reason to only allow mounted/present? (which is still better than enforced `mounted`!! :wink:)

Reply:
Ok, I will prefix and move the variables.
I have never used the states: `unmounted ` or `absent `with ansible.
Anyway, by default the role `nfs_client ` use the `mounted` option, this is not changing here.
I propose to use the `present` option uniquely to be able to run this unitary tests, this does not impact the role installation.

Reply:
Corrected in commit f28b555